Cargo.toml

```diff
@@ -38,6 +38,7 @@ daggy = { version = "0.8", features = [ "serde" ] }
 dialoguer = "0.11"
 diesel = { version = "2", features = ["postgres", "chrono", "uuid", "serde_json", "r2d2"] }
 diesel_migrations = "2"
+enum-iterator = "2.1"
```

Comment:
Our convention is to use only the major version here (except if it's 0, then we use the major and minor version) - that way it should usually remain semver compatible (there's lots more to it though...).